### PR TITLE
Modified privacy policy page to the current version of EvalAI

### PR DESCRIPTION
**Pull request by Sanji515**
- Added the scrollbar from the EvalAI project

- Toggle the show/hide for the `scroll to top` button when scroll is greater or lesser than 180 respectively

- Changed the highlight color of the content title and also changed css for the texts to look more better

**<p align="center">Screenshot 1</p>**

Screenshot from 2019-06-14 21-55-32

**<p align="center">Screenshot 2</p>**

Screenshot from 2019-06-14 21-53-24

### CODE REVIEW

File: src/app/components/privacy-policy/privacy-policy.component.scss [hunk 1]
```diff
@@ -36,21 +34,21 @@
       }
 
       li.scroll-selected {
-        color: $red-light;
```
Comment (Shekharrajak) [inline src/app/components/privacy-policy/privacy-policy.component.scss:40]: you can directly change the red-light variable. Also I think bit darker color will be better to read the text.

**Review by Shekharrajak — COMMENTED** [review]
I think this should be top of https://github.com/Cloud-CV/EvalAI-ngx/pull/111

File: src/app/components/privacy-policy/privacy-policy.component.scss (hunk 1 above)
Comment (Sanji515) [reply src/app/components/privacy-policy/privacy-policy.component.scss:40]: @Shekharrajak done

**Comment by Shekharrajak** [comment]
Please update the branch and resolve conflicts.

**Comment by Sanji515** [comment]
@Shekharrajak Please review

After merging, `scrollIntoView` was not working properly. The offset of navbar is not included so I've used `scrollTo` in order to give offset of the navbar to scroll at the right element

**Comment by Sanji515** [comment]
@Shekharrajak Please review

**Review by Shekharrajak — COMMENTED** [review]
It looks good for now. But if you observe it properly then you will find that UI is not that much attractive and smooth.

Spaces can be removed (like around nav bar, footer, left sidebar, ...) You can try out the demo URL in different devices.

*

**Comment by Sanji515** [comment]
> Spaces can be removed (like around nav bar, footer, left sidebar, ...)

Current navbar's height is different from the navbar in the current version and also same for the footer, it will be changed and also I left the spaces around sidebar to make the main content to align in the center to look more better

please let me know what are your thoughts, I can change if you want

**Comment by Sanji515** [comment]
> You can try out the demo URL in different devices

I have checked in the browser for different devices, a/c to me it's working perfectly, please check if it's looks good to you as well

**Comment by lunayach** [comment]
@Sanji515 could you add some unit tests so that we merge this?

File: src/app/components/privacy-policy/privacy-policy.component.spec.ts [hunk 2]
```diff
@@ -68,4 +79,24 @@ describe('PrivacyPolicyComponent', () => {
...
+      expect(component.highlightNav).toHaveBeenCalled();
+    });
+  });
+
+  it('should have called the highlight functions when scroll selected element is called', () => {
+    const scrollSelected = de.query(By.css('.scroll-selected'));
+    scrollSelected.nativeElement.click();
+
+    fixture.whenStable().then(() => {
+      expect(component.highlightNav).toHaveBeenCalled();
```
Comment (lunayach) [inline src/app/components/privacy-policy/privacy-policy.component.spec.ts]: Repeated.

File: src/app/components/privacy-policy/privacy-policy.component.spec.ts [hunk 3]
```diff
@@ -68,4 +79,24 @@ describe('PrivacyPolicyComponent', () => {
...
+      expect(component.highlightSectionTitle).toHaveBeenCalled();
```
Comment (lunayach) [inline src/app/components/privacy-policy/privacy-policy.component.spec.ts:89]: This could have been written along with the above two "expect"s.

Comment (Sanji515) [reply src/app/components/privacy-policy/privacy-policy.component.spec.ts:89]: @lunayach  done

**Review by lunayach — APPROVED** [review]
Looks good to me.

**Review by Shekharrajak — APPROVED** [review]

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**